### PR TITLE
Dev friendly amp improvements and windows support

### PR DESCRIPTION
**Pull request by lacinoire**
- Windows-compatible default values for paths to sources/classes.
- No longer add junit 4 separately to dependencies (this was creating failures when the path to the project contains a space)
- Contributing various small improvements that make --dev-friendly amplified test cases clearer to read:
  - Assertions & their arguments are removed completely
  - Comments & consistent naming for NullLiteralAmplifier, JUnit5 Exception-Assertiongenerator, MethodCallRemover

This required to change the test cases including CommentTest#testAmplifierComments, hopefully making it no longer flaky.
The amplification is still non-deterministic, though the test now only checks for presence of the comment, not the exact change that was made in the test.

### CODE REVIEW

**Comment by lacinoire** [comment]
Might address #1001

File: dspot/src/main/java/eu/stamp_project/dspot/common/configuration/InitializeDSpot.java [hunk 1]
```diff
@@ -150,14 +149,15 @@ public String completeDependencies(UserInput configuration, AutomaticBuilder aut
...
         // TODO checks this. Since we support different Test Support, we may not need to add artificially junit in the classpath
-        if (!dependencies.contains("junit" + File.separator + "junit" + File.separator + "4")) {
-            dependencies = Test.class
-                    .getProtectionDomain()
-                    .getCodeSource()
-                    .getLocation()
-                    .getFile() +
-                    AmplificationHelper.PATH_SEPARATOR + dependencies;
-        }
+//        if (!dependencies.contains("junit" + File.separator + "junit" + File.separator + "4")) {
```
Comment (danglotb) [inline dspot/src/main/java/eu/stamp_project/dspot/common/configuration/InitializeDSpot.java:152]: If this code is no longer useful, please remove it completely

**Review by danglotb — APPROVED** [review]

**Comment by danglotb** [comment]
Hi @lacinoire
thank you very much for your contributions! It is great :smile: 
Could you address my comment? I'll merge afterward.

**Comment by lacinoire** [comment]
Hey @danglotb, 
Thank you for the review  😊
Good point! I removed the commented out code now 🙂

**Pull request merged**